repo-issues/lib/data-source.js

```diff
@@ -0,0 +1,97 @@
+const http = require('request-promise-native')
+const moment = require('moment')
+const octokit = require('@octokit/rest')()
+
+const show_detaiils_threshold = 5
+const organisation = 'novoda'
```

Comment:
These two constants could be extracted if we wish to show repos from other organisations or increase the threshold to show/hide details